packages/crypto/CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # @turnkey/crypto
 
+## 2.1.0
+
+### Minor Changes
+
+- 33e8e03: Reorganize into two subparts:
+
+  - `crypto.ts`: core cryptography utilities
+  - `turnkey.ts`: Turnkey-specific cryptography utilities
+
+  Add `verifyStampSignature` method:
+
+  - See in-line code docs for more details + example of usage
+  - This is useful for checking the validity of a stamp (signature) against the request body
```

Comment:
This looks a bit weirdly formatted. Let's link to the PR number (instead of the git sha) and cleanup the bullets so it's one list.